### PR TITLE
Add ability to select a file in project.json

### PR DESCRIPTION
**Pull request by kevinpschaaf**
The selected file in the file tabs is selected based on the selected file in the project config only when loading a new project.

Fixes https://github.com/google/playground-elements/issues/303

### CODE REVIEW

File: src/playground-project.ts [hunk 1]
```diff
@@ -51,6 +51,20 @@ const generateUniqueSessionId = (): string => {
...
 };
 
+declare global {
+  interface HTMLElementEventMap {
+    filesChanged: FilesChangedEvent;
+  }
+}
+
+export class FilesChangedEvent extends Event {
+  projectLoaded: boolean;
```
Comment (aomarks) [inline src/playground-project.ts:61]: WDYT about a separate `projectLoaded` event? Seems like that could be a little simpler.

**Review by aomarks — CHANGES_REQUESTED** [review]
Looks good, just a thought about whether we should have 2 events instead of a bit.

Also, could you add a CHANGELOG entry, and update these two sections of the README to add `selected`?

https://github.com/google/playground-elements#option-1-inline-scripts
https://github.com/google/playground-elements#option-2-json-configuration

**Comment by aomarks** [comment]
Also looks like it needs an `npm run format`

**Review by aomarks — APPROVED** [review]

File: src/playground-project.ts (hunk 1 above)
Comment (kevinpschaaf) [reply src/playground-project.ts:61]: The current work of `filesChanged` needs to happen for both "project loaded" and "project files changed" cases, so it'd be a bit more complex to listen both events and dedupe them correctly. Leaving as is per discussion.

**Review by AndrewJakubowicz — APPROVED** [review]
Superb!

This also provides a great way to handle the removal of files between lit.dev tutorial steps.

**Pull request merged**